content/zh/docs/concepts/observability-primer.md

```diff
@@ -0,0 +1,100 @@
+---
+title: 可观测性入门
+description: 可观测性的核心概念
+weight: 9
+cSpell:ignore: webshop
+default_lang_commit: e771c886739c4847b332b74f24b09d2769aab875
+---
+
+## 什么是可观测性？ {#what-is-observability}
+
+可观测性让你能够从外部理解一个系统，它允许你在不了解系统内部运作的情况下，对该系统提出问题。更重要的是，
+它能帮你轻松排查和处理新出现的问题，也就是所谓的"未知的未知"。它还能帮你回答"为什么会发生这种情况？"这样的问题。
+
+要对你的系统提出这些问题，你的应用程序必须进行适当的插桩。也就是说，应用程序代码必须能够发出[信号](/docs/concepts/signals/)
+例如
+[追踪](/docs/concepts/signals/traces/)、
```

Comment:
```suggestion
例如[链路](/docs/concepts/signals/traces/)、
```
trace is a noun here, how about to use 链路 instead of 追踪? Seems you can replace it from top to bottom.

Reply:
好的，感谢review